### PR TITLE
Feature/qa 429 test not two agents for net

### PR DESCRIPTION
**Pull request by rpromyshlennikov**

### CODE REVIEW

File: mos_tests/neutron/python_tests/test_ban_dhcp_agent.py [hunk 1]
```diff
@@ -510,6 +518,77 @@ def _killing_cycle(
...
+        # Fixture init from method self._prepare_openstack_state
+        # Get all dhcp agents
+
+        all_agents = self.os_conn.list_all_neutron_agents(agent_type='dhcp',
+                                                          filter_attr='host')
+        curr_agents = self.os_conn.get_node_with_dhcp_for_network(
+            net_id=self.net_id)
+
+        # determine primary controller
+        leader_node = self.env.leader_controller.data['ip']
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_ban_dhcp_agent.py]: May be `leader_ip`?

File: mos_tests/neutron/python_tests/test_ban_dhcp_agent.py [hunk 2]
```diff
@@ -588,3 +667,150 @@ def test_reschedule_dhcp_agents(self):
...
                                             new_ports_ids))
         assert sorted(ports_ids) == sorted(new_ports_ids), err_msg
+
+
+class TestBanDHCPAgentWithSettings(TestBaseDHCPAgent):
+    """Test with preparation of neutron service."""
+    @staticmethod
+    def _change_network_count_on_dhcp_agent(remote, value,
+                                            param=None, path=None):
+        """Change some parameter in config to new value.
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_ban_dhcp_agent.py]: Method name and docstring are mismatched

File: mos_tests/neutron/python_tests/test_ban_dhcp_agent.py [hunk 3]
```diff
@@ -479,9 +486,10 @@ def _killing_cycle(
         agents_networks = [net['id'] for agent_id in agents_ids for net in
                            self.os_conn.get_networks_on_dhcp_agent(agent_id)]
         # determine primary controller
-        leader_node = self.env.leader_controller.data['ip']
+        leader_node_ip = self.env.leader_controller.data['ip']
```
Comment (TimurNurlygayanov) [inline mos_tests/neutron/python_tests/test_ban_dhcp_agent.py:489]: primary_node_ip?

Comment (rpromyshlennikov) [inline mos_tests/neutron/python_tests/test_ban_dhcp_agent.py:489]: primary != leader in this context

**Pull request merged**